Wake On Lan/Modules/AboutScreen/Tests/AboutScreenTests/AboutScreenInteractorTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+
+@testable import AboutScreen
+@testable import WOLResources
+import WOLResourcesMock
+
+final class AboutScreenInteractorTests: XCTestCase {
+
+    var sut: AboutScreenInteractor!
```

Comment:
```suggestion
    var aboutScreenInteractor: AboutScreenInteractor!
```

Reply:
sut means system under test, and as far as I know it is common practice